nibabel/nifti1.py

```diff
@@ -528,6 +528,15 @@ def from_fileobj(klass, fileobj, size, byteswap):
             extensions.append(ext)
         return extensions
 
+# Temporary hack to avoid huge git diff
+Nifti1Extension.extension_codes = Nifti1Extensions.extension_codes
+
+class DeprecatedExtensionCodes(Nifti1Extensions):
+    @np.deprecate_with_doc("Use Nifti1Extensions.extension_codes")
+    def __getitem__(self, *args, **kwargs):
+        return super(DeprecatedExtensionCodes, self).__getitem__(*args, **kwargs)
+extension_codes = DeprecatedExtensionCodes.extension_codes
+
```

Comment:
Perhaps I should pull this commit out and do a PR directly to `master`, to describe the issue and the solution I'm using.